build.sbt

```diff
@@ -47,6 +47,6 @@ lazy val zapAutomation = Project(appName, file("."))
     makePublicallyAvailableOnBintray := true,
     scalaVersion := "2.11.12",
     libraryDependencies ++= compileDependencies ++ testDependencies,
-    crossScalaVersions := Seq("2.11.12"),
+    crossScalaVersions := Seq("2.11.12", "2.12.7"),
```

Comment:
We could use `2.12.10` which is a little newer?

Reply:
👍 I also took SBT to 1.3.8